gateways/adyen/request_builder_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build unit
```

Comment:
How does this differ from the line below? Can we just use one of them?

Reply:
Its the new format but the formatter is adding both for a few versions
https://stackoverflow.com/questions/10646531/golang-conditional-compilation/67937234#67937234